src/Tool-DependencyAnalyser-Tests/DAMessageSendAnalyzerTest.class.st

```diff
@@ -13,6 +13,16 @@ DAMessageSendAnalyzerTest >> setUp [
 	analyzer := DAMessageSendAnalyzer on: 'Tool-DependencyAnalyser-Tests-Data'
 ]
 
+{ #category : #tests }
+DAMessageSendAnalyzerTest >> testRPackage [
+
+	self assert: (analyzer rPackage isKindOf: RPackage).
+	analyzer := DAMessageSendAnalyzer on: 'Unloaded-Dummy-Package'.
+	self
+		assert: (analyzer rPackage isKindOf: RPackage)
+		description: 'It test that rPackage still answer a RPackage even when the analyzer was instantiated with an unexisting or unloaded package'
```

Comment:
I'm not fond of tests doing isKindOf, but ok.

This kind of tests prevent future extensibility. Anyways, if required the test can be removed in the future.

Reply:
A stronger test should be to send a message to the `analyzer rPackage` asserting expected classes or methods in the package

```smalltalk
self assert: analyzer rPackage definedClasses equals: ...
```

That asserts that the returned object works correctly, catches the bug too, and leaves the door open to extensions :)

Reply:
I don't understand, the test is gone, is that on purpose?

Reply:
I re-added the missing methods. 
Added as testRPackageIsAnsweredWithLoadedPackage